### PR TITLE
Add static legend (addresses #2568 )

### LINKED ISSUE #2568: Add documentation for static legend, a new dashboard feature
**PR:** _Provide PR URL(s) for this feature (if available)_

_Describe the new feature here._
We are introducing a feature to add a static legend to Graph (overlaid, stacked) and Band Plot. 
static1
static

##### Relevant URLs
- _Provide relevant URLs (issues, community threads, existing doc pages, etc.)_
The [epic](https://app.zenhub.com/workspaces/applicationui-5e3b05772922e316b3a210a4/issues/influxdata/ui/14) for this feature

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #2568

### CODE REVIEW

File: content/influxdb/v2.0/visualize-data/visualization-types/band.md [hunk 1]
```diff
@@ -155,8 +155,9 @@ To view **Band** controls, click **{{< icon "gear" >}} Customize** next to the v
 
 ###### Legend
 
-- **Legend Orientation**: Select the orientation of the legend that appears upon hover:
+- **Legend Orientation**: Select the orientation of the legend:
   - **Horizontal**: Select to display the legend horizontally.
   - **Vertical**: Select to display the legend vertically.
 - **Opacity**: Adjust the legend opacity using the slider.
 - **Colorize Rows**: Select to display legend rows in colors.
+- **Static Legend**: Select **Hide** to show the legend only upon hover. Select **Show** to always show the legend.
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/visualization-types/band.md:163]:
```suggestion
- **Static Legend**: Select **Hide** to hide the legend or if the **Hover Legend Show** option is selected, show the legend only upon hover. Select **Show** to always show the legend.
```

File: content/influxdb/v2.0/visualize-data/visualization-types/band.md [hunk 2]
```diff
@@ -155,8 +155,9 @@ To view **Band** controls, click **{{< icon "gear" >}} Customize** next to the v
 
 ###### Legend
 
-- **Legend Orientation**: Select the orientation of the legend that appears upon hover:
+- **Legend Orientation**: Select the orientation of the legend:
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/visualization-types/band.md:158]:
```suggestion
- **Orientation**: Select the orientation of the legend:
```

File: content/influxdb/v2.0/visualize-data/visualization-types/graph-single-stat.md [hunk 3]
```diff
@@ -101,11 +101,13 @@ the visualization dropdown.
   Choose **Background** for the background of the graph to change color based on the configured thresholds.
 
 ###### Legend
-- **Legend Orientation**: Select the orientation of the legend that appears upon hover:
+- **Legend Orientation**: Select the orientation of the legend that appears:
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/visualization-types/graph-single-stat.md:104]:
```suggestion
- **Orientation**: Select the orientation of the legend that appears:
```

File: content/influxdb/v2.0/visualize-data/visualization-types/graph-single-stat.md [hunk 4]
```diff
@@ -101,11 +101,13 @@ the visualization dropdown.
...
   - **Horizontal**: Select to display the legend horizontally.
   - **Vertical**: Select to display the legend vertically.
 - **Opacity**: Adjust the legend opacity using the slider.
 - **Colorize Rows**: Select to display legend rows in colors.
+- **Static Legend**: Select **Hide** to show the legend only upon hover. Select **Show** to always show the legend.
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/visualization-types/graph-single-stat.md:109]:
```suggestion
- **Static Legend**: Select **Hide** to hide the legend or if the **Hover Legend Show** option is selected, show the legend only upon hover. Select **Show** to always show the legend.
```

File: content/influxdb/v2.0/visualize-data/visualization-types/graph.md [hunk 5]
```diff
@@ -76,11 +76,13 @@ the visualization dropdown.
   - **Stacked**: Display graph lines stacked on top of each other.
 
 ###### Legend
-- **Legend Orientation**: Select the orientation of the legend that appears upon hover:
+- **Legend Orientation**: Select the orientation of the legend that appears:
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/visualization-types/graph.md:79]:
```suggestion
- **Orientation**: Select the orientation of the legend that appears:
```

File: content/influxdb/v2.0/visualize-data/visualization-types/graph.md [hunk 6]
```diff
@@ -76,11 +76,13 @@ the visualization dropdown.
...
   - **Horizontal**: Select to display the legend horizontally.
   - **Vertical**: Select to display the legend vertically.
 - **Opacity**: Adjust the legend opacity using the slider.
 - **Colorize Rows**: Select to display legend rows in colors.
+- **Static Legend**: Select **Hide** to show the legend only upon hover. Select **Show** to always show the legend.
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/visualization-types/graph.md:84]:
```suggestion
- **Static Legend**: Select **Hide** to hide the legend or if the **Hover Legend Show** option is selected, show the legend only upon hover. Select **Show** to always show the legend.
```

**Review by kelseiv — APPROVED** [review]

**Comment by daviddkkim** [comment]
@noramullen1  Would you like any images or GIFs for this?

**Comment by noramullen1** [comment]
> @noramullen1 Would you like any images or GIFs for this?

We don't use images for these docs, thank you though! :)

**Pull request merged**